src/logic.js

```diff
@@ -162,12 +146,6 @@ define([
                     replacePathInfo(this._xpathParser.parse(to), path);
                 }
             }
-            for (i = 0; i < hashtags.length; i++) {
-                path = hashtags[i];
-                if (path.toHashtag() === from) {
-                    replacePathInfo(this._xpathParser.parse(to), path);
-                }
-            }
```

Comment:
This change was not strictly necessary. It's DRYing things up a bit, along with the change above on line 130.